CONTRIBUTORS.txt

```diff
@@ -259,5 +259,5 @@ In chronological order:
   * Improve contribution guide
   * Add ``HTTPResponse.geturl`` method to provide ``urllib2.urlopen().geturl()`` behavior
 
-* [Your name or handle] <[email or website]>
-  * [Brief summary of your changes]
```

Comment:
Note that you need to leave this here for future contributors. :)

Reply:
Doh! Fixed.